test/framework/test_description.go

```diff
@@ -45,12 +45,6 @@ func (t TestDescription) Serial() TestDescription {
 	return t.newLabel("SERIAL")
 }
 
-// Disruptive labels a test as disruptive.
-// This kind of test should run with care.
-func (t TestDescription) Disruptive() TestDescription {
-	return t.newLabel("DISRUPTIVE")
```

Comment:
The concept for a disruptive test is baked in testmachinery: see https://github.com/search?q=repo%3Agardener%2Ftest-infra%20DISRUPTIVE&type=code.

I recall discussions with @dguendisch and @hendrikKahl that these disruptive tests maybe no longer make sense!? Can we use the opportunity to discuss again if we need disruptive tests at all? If that's the case, we can have an issue in gardener/test-infra to drop the concept for disruptive tests. 

I recall with @dimitar-kostadinov we had conceptual issues for registry-cache TM tests whether to make the CA rotation test disruptive or not.